acquia/cli
# CLI-871 | testConfigurePlatformEmail doesn't check predictions, generates cruft

**Motivation**
Adjusts tests re: configuring Platform Email so that predictions are correctly checked and the mocked local file system is being used (instead of adding extra files to the repo)

**Testing steps**

1. Follow the [contribution guide](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#building-and-testing) to set up your development environment or [download a pre-built acli.phar](https://github.com/acquia/cli/blob/HEAD/CONTRIBUTING.md#automatic-dev-builds) for this PR.
2. Clear the kernel cache to pick up new and changed commands: `./bin/acli ckc`
3. Run PHPUnit to verify that there are no more files generated while running tests and that tests still pass!
